scripts/lib/CIME/SystemTests/system_tests_compare_two.py

```diff
@@ -161,28 +193,38 @@ def run_phase(self, success_change=False):  # pylint: disable=arguments-differ
         Runs both phases of the two-phase test and compares their results
         If success_change is True, success requires some files to be different
         """
+        first_phase = self._case1.get_value("RESUBMIT") == 1
+        run_type = self._case1.get_value("RUN_TYPE")
 
         # First run
-        logger.info('Doing first run: ' + self._run_one_description)
-        self._activate_case1()
-        run_type = self._case1.get_value("RUN_TYPE")
-        self.run_indv(suffix = self._run_one_suffix)
+        if not self._multisubmit or first_phase:
```

Comment:
@billsacks This is tested by the ERR test in cime_developer. If you want to enhance the unittests for system_tests_compare_two, I'll leave that to you. I had a hard time figuring out what was going on in there.